spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_graph_target_host_spec.rb

```diff
@@ -1,4 +1,8 @@
+require_relative 'ovirt_refresher_spec_common'
+
 describe ManageIQ::Providers::Redhat::InfraManager::Refresh::Refresher do
+  include OvirtRefresherSpecCommon
+
   describe ManageIQ::Providers::Redhat::InfraManager::Refresh::Refresher do
```

Comment:
@borod108 why are these describe blocks duplicated?

Reply:
For no good reason I can think off.